android/src/main/java/expo/modules/inapppurchases/BillingManager.java

```diff
@@ -141,7 +141,7 @@ public void purchaseItemAsync(final String skuId, @Nullable final ReadableArgume
     ReadableArguments d = details != null ? details : new MapArguments();
     String oldPurchaseToken = d.getString("oldPurchaseToken");
     ReadableArguments accountIdentifiers = d.getArguments("accountIdentifiers");
-    Boolean isVrPurchaseFlow = d.getBoolean("isVrPurchaseFlow");
+    // Boolean isVrPurchaseFlow = d.getBoolean("isVrPurchaseFlow");
```

Comment:
Is this no longer supported, is there another property which replaces it?

Reply:
I have no idea. Just got it from a comment. Do you know if this property is useful for something related to our project?

Reply:
https://github.com/expo/expo/issues/23920#issuecomment-1693813275

Reply:
It's not I guess we can add it back if needed